release_tools/README.adoc

```diff
@@ -55,7 +99,9 @@ The last commit before the release has to have the `openscap-<version>` message
 +
 In a clean copy of the release branch run `cmake .. && make package_source` in the `build` directory.
 +
-Create new GitHub release with the name of the vesrion (e.g. 1.3.2) being released.
+Check the archive for suspicious files, prevent situation horse birth!
```

Comment:
Even if it is a metaphor that I'm unaware of, I'd say it is a bit inappropriate. Let's stay away from procreation allegories.

Reply:
You haven't changed this part. Do you think that this wording is appropriate?

Reply:
I think that it's important to check the generated archive if there are files that shouldn't be there. It can raise questions or cause downstream builds or tests to fail.

Reply:
Of course! But "horse birth"?

Reply:
It's a situation when in past in our team a released tarball of `sectool` contained a video showing a horse giving birth.

Reply:
Cool. Nice story to tell in a pub on our next in-person meeting, can't wait to hear it. Now, please, remove this part from the upstream release guideline.

Reply:
it already wasn't there